docs/_docs/upgrading.md

```diff
@@ -6,5 +6,4 @@ permalink: /docs/upgrading/
 
 Upgrading from an older version of Jekyll? Upgrading to a new major version of Jekyll (e.g. from v2.x to v3.x) may cause some headaches. Take the following guides to aid your upgrade:
```

Comment:
OK, will update. Am I correct in that this is just another entry inside of `docs/redirects`?